inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/stream/InlongStreamServiceImpl.java

```diff
@@ -514,12 +569,31 @@ public Boolean updateWithoutCheck(InlongStreamRequest request, String operator)
         // update stream fields
         updateField(groupId, streamId, request.getFieldList());
         // update stream extension infos
-        saveOrUpdateExt(groupId, streamId, request.getExtList());
+        List<InlongStreamExtInfo> extList = request.getExtList();
+        saveOrUpdateExt(groupId, streamId, extList);
 
         LOGGER.info("success to update inlong stream without check for groupId={} streamId={}", groupId, streamId);
         return true;
     }
 
+    /**
+     * Pack extended attributes into ExtParams 
+     *
+     * @param request the request
+     * @return the packed extParams
+     */
+    private String packExtParams(InlongStreamRequest request) {
+        InlongStreamExtParamBuilder builder = InlongStreamExtParam.builder();
```

Comment:
It is recommended to use `CommonBeanUtils.copyProperties(request, InlongStreamExtParam::new, true);`

Reply:
The `org.apache.inlong.manager.dao.entity.InlongStreamEntity`
does not contain the same fields, json must be compressed into extParams, so `CommonBeanUtils.copyProperties(request, InlongStreamExtParam::new, true)` cannot be used;

Reply:
ok, done